content/reference-data-api.md

```diff
@@ -3006,6 +3212,190 @@ Response:
 
 
 
+## Get Account Transaction Stats ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/accountStats.js "Source")
+
+Retrieve daily summaries of transaction activity for an account. _(New in [v2.1.0][].)_
+
+<!--<div class='multicode'>-->
+
+*REST*
+
+```
+GET /v2/accounts/{:address}/stats/transactions
+```
+
+<!--</div>-->
+
+[Try it! >](data-api-v2-tool.html#get-account-transaction-stats)
+
+This method requires the following URL parameters:
+
+| Field    | Value  | Description |
+|----------|--------|-------------|
+| :address | String | Ripple address to query |
```

Comment:
Cool. It might just be my eyes, but using monospace, bold, etc, seems more important for PDFs than HTML. Something to think about for the future.